src/patterns/gen-ai/aws-llama-index-data-loader/README.md

```diff
@@ -0,0 +1,134 @@
+| existingDashboard               | [aws_cloudwatch.Dashboard](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_cloudwatch.Dashboard.html)                                | ![Optional](https://img.shields.io/badge/optional-4169E1) | Existing dashboard to be used by the construct. **Mutually exclusive** with ```dashboardName``` - only one should be specified.                                                                                                                                                                                                                                                           |
+| dashboardName | string | ![Optional](https://img.shields.io/badge/optional-4169E1) | A name for the dashboard which will be created. If not provided, the construct will create a new dashboard named 'AossMetricsDashboard'. **Mutually exclusive** with ```existingDashboard``` - only one should be specified.                                                                                                                                                                                                                                                            |
+
+## Pattern Properties
+
+| **Name**                     | **Type**                                                                                                                  | **Description**                                                                                                                                                                |
+| :----------------------------- | :-------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| dashboard                          | [aws_cloudwatch.Dashboard](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_cloudwatch.Dashboard.html)                                     | The CloudWatch Dashboard used by the construct (whether created by the construct or provided by the client)                                                                                     |
+
+## Methods
+
+N/A
+
+## Default properties
+
+Out-of-the-box implementation of the construct without any override will set the following defaults:
```

Comment:
Need to provide the default properties (memory limit,...) without any override from the user

Reply:
fixed 97a94356b0cd6484195e73760c92c740b30c57c9